### PR TITLE
Travis CICD

### PR DESCRIPTION
**Pull request by Lucy911**
Travis running katlas docker images for test and push to docker hub when test is success.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,17 +19,44 @@ before_script:
...
+  - docker build --no-cache -f Dockerfile -t katlas/katlas-service:${TRAVIS_COMMIT_SHORTID} .
+  - docker images katlas/katlas-service:${TRAVIS_COMMIT_SHORTID}
   - cd $TRAVIS_BUILD_DIR/app
   - yarn test --coverage
-  - docker build --no-cache -f Dockerfile -t katlas-browser:${TRAVIS_COMMIT} .
+  - docker build --no-cache -f Dockerfile -t katlas/katlas-browser:${TRAVIS_COMMIT_SHORTID} .
+  - docker images katlas/katlas-browser:${TRAVIS_COMMIT_SHORTID}
 
 after_success:
+  - sleep 5s
```
Comment (fzxu) [inline .travis.yml]: is it needed?

Comment (Lucy911) [reply .travis.yml]: Should not needed. I'll remove it.

**Comment by Lucy911** [comment]
@srivathsanvc Thank you Srivathsan. Will do shortly.

File: .travis.yml [hunk 2]
```diff
@@ -19,17 +19,45 @@ before_script:
...
+  - docker build --no-cache -f Dockerfile -t katlas/katlas-browser:${TRAVIS_COMMIT_SHORTID} .
+  - docker images katlas/katlas-browser:${TRAVIS_COMMIT_SHORTID}
 
 after_success:
   - cd $TRAVIS_BUILD_DIR/service
   - bash <(curl -s https://codecov.io/bash)
   - cd $TRAVIS_BUILD_DIR/app
   - bash <(curl -s https://codecov.io/bash) -f coverage/coverage-final.json
+  - cd $TRAVIS_BUILD_DIR/service
+  #- go run server.go 2> output.out &
```
Comment (fzxu) [inline .travis.yml]: you can remove the comment outed codes.

File: .travis.yml [hunk 3]
```diff
@@ -19,17 +19,45 @@ before_script:
...
+  - echo $statusCode
+  - |
+    if [ $statusCode -ne "200" ]; then
+        echo "Integration Test - Fail!";
+    else
+        echo "Integration Test - Success!";
+        if [["$TRAVIS_BRANCH" == "master" ]]; then
+            # Login to docker hub
+            echo "$DOCKER_PASSWORD" |docker login -u $DOCKER_USERNAME --password-stdin
+            # echo "$DOCKER_PASSWORD" |docker login -u $DOCKER_USERNAME --password-stdin >/dev/null 2>&1
```
Comment (fzxu) [inline .travis.yml]: remove the comment outed code.

**Pull request merged**